src/main/java/org/time2java/tRussianBank/domain/User.java

```diff
@@ -0,0 +1,25 @@
+package org.time2java.tRussianBank.domain;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+/**
+ * Created by time2die on 08.11.16.
+ */
+@Data
+@Builder
+@EqualsAndHashCode
+public class User {
+    String name ;
+    String vkID ;
+    String city ;
```

Comment:
\[Checkstyle\] ERROR: Variable 'city' must be private and have accessor methods\.